package.json

```diff
@@ -18,14 +18,16 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.3",
+    "@types/uuid": "^9.0.1",
```

Comment:
Please don't do this. I have mentioned this in like 3 other separate P.R. You should know this by now.